### DISCUSSION TITLE
Display inline images in Jupyter notebook

### DISCUSSION THREAD

**Discussion by rupert-madden-abbott** [Textualize/rich#1384] [Q&A]
If I render an inline image using Markdown directly in Jupyter then I get the image display inline on the page.

If I render via rich, then I get an invalid link to the image (clicking it gives a 404).

Is there a way to get rich to display the image inline when it is used inside Jupyter or do I need to detect that I am running in Jupyter and handle this myself?

**Comment by willmcgugan** [comment]
The Jupyter support in Rich replicates exactly what it would print in the terminal, which doesn't support images.
